doc/dev/design/workflows/storage_devices_and_osds.rst

```diff
@@ -0,0 +1,304 @@
+==============================================
+Storage devices and OSD’s management workflows
+==============================================
+The cluster storage devices are the physical storage devices installed in each of the cluster’s hosts. We need to execute different operations over them and also to retrieve information about physical features and working behaviour.
+The basic use cases we have in this area are:
+
+- `1. Retrieve device information. Inventory`_
+- `2. Add OSDs`_
+- `3. Remove OSDs`_
+
+1. Retrieve device information. Inventory
+=========================================
+We must be able to review what is the current state and condition of the cluster storage devices. We need the identification and features detail(including ident/fault led on/off capable) and if the device is used or not as an OSD/DB/WAL device.
```

Comment:
I think we should also put a big caveat in here, that the possibility to activate ident/fault LEDs is heavily dependent on the hardware used and may not be generally applicable/available.

Reply:
yes exactly. We should know in advance and show this info, if we have this capability or not in certain device
In this moment you can try to switch the led on in all the devices, it raises an error if we coont do the operation, but by default  is always possible to execute in all the devices

Reply:
That's a very good point, which could actually be considered a bug, even though I'm not quite sure how to resolve it. The dashboard should not give the impression that some feature is available if it actually isn't. This should be tracked as a usability issue under this epic: [mgr/dashboard: Usability Improvements](https://tracker.ceph.com/issues/47072)

Reply:
@LenzGr : 
https://tracker.ceph.com/issues/47858